learning_resources_search/serializers.py

```diff
@@ -311,3 +386,26 @@ def serialize_for_deletion(opensearch_object_id):
         dict: the object deletion data
     """
     return {"_id": opensearch_object_id, "_op_type": "delete"}
+
+
+def serialize_content_file_for_bulk(content_file_obj):
+    """
+    Serialize a content file for bulk API request
+
+    Args:
+        content_file_obj (ContentFile): A content file for a course
+    """
+    return {
+        "_id": gen_content_file_id(content_file_obj.key),
+        **serialize_content_file_for_update(content_file_obj),
```

Comment:
Some `content` values for ContentFiles are so large that they might [cause errors when trying to index](https://github.com/mitodl/open-discussions/issues/3881) - might want to truncate them as done in https://github.com/mitodl/open-discussions/pull/3917.  

Maybe we shouldn't show the entire `content` field in the API output either to avoid problems when it is excessively large? Or exclude it from the output altogether?

Reply:
PS I dont have any contentfiles over 10 MB locally yet, the biggest is 4.5 MB from OCW course https://live-qa.ocw.mit.edu/courses/hst-508-quantitative-genomics-fall-2005/

http://localhost:8063/api/v1/content_file_search/?q=%22CDNA-Pos%20%20Description%20%20%20Allele1%22 will return it if imported.

Reply:
So that actually looks fine in the ui since the default api view does not wrap lines

Reply:
<img width="1689" alt="Screenshot 2023-11-03 at 9 31 05 AM" src="https://github.com/mitodl/mit-open/assets/1934992/f2fbc05e-1790-435a-9461-15c94c314c12">